python/README.md

```diff
@@ -0,0 +1,18 @@
+# How to build/publish
+
+Follow the guide here: https://help.github.com/en/articles/configuring-docker-for-use-with-github-package-registry
+
+
+# Dev/testing image
+
+Build:
+
+```sh
+docker-compose build ddtracepy-dev
```

Comment:
Can we use `dd-trace-py-dev`?
We never ever use `ddtracepy` as a name so it's just going to be… confusing and not handy.